### DISCUSSION TITLE
When can we expect the next MIMIC?

### DISCUSSION THREAD

**Discussion by arielhasidim** [MIT-LCP/mimic-code#1566] [MIMIC-General] [answered]
I'm interested in data relating to COVID-19. 
Is there any forecast for the next MIMIC-V with hospitalization after 2019?

**Comment by alistairewj** [comment]
Probably not until 2024? We need 1 year post-discharge followup.

**Reply by arielhasidim** [reply]
Thank you @alistairewj 
Maybe I misunderstood, but it was to my knowledge that MIMIC-IV covered 2008-2019. I'm specifically interested in ICU stays between 2020-2022, which by now should be done with the 1-year post-discharge follow-up. 
Can you please clarify?

**Reply by alistairewj** [reply]
The last discharge in 2022 could have followup through the end of 2023. Primarily relating to out of hospital mortality.

**Reply by arielhasidim** [reply]
Okay, this makes sense - the end of 2022 will end follow-up at the end of 2023.

To clarify, is there any plan to release a new version of MIMIC, say MIMIC "V," in 2024?

**Reply by alistairewj** [reply, marked as answer]
I don't think so, I think it will be MIMIC-IV v3.0
